www/cgi-bin/create-log.sh

```diff
@@ -0,0 +1,29 @@
+#Copyright 2018 - 2019 Andrei Khodko
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License
+
+#!/bin/sh
+
+FILENAME=$(./etc/trik/log_manager.sh --all)
+FILESIZE=$(stat -c%s "$FILENAME")
+
+cat << EOF
```

Comment:
Add support for notifier to check on desktop version

Reply:
`/notifyThenKill.sh $(basename -- "$0") $$`